src/js/components/ProductManageMenu.js

```diff
@@ -0,0 +1,120 @@
+import { SELECTOR } from '../constants/selector.js';
+import { $ } from '../utils/dom.js';
+import { validateProductName, validateProductPrice, validateProductQuantity } from '../utils/validation.js';
+
+/* eslint-disable class-methods-use-this */
+export default class ProductManageMenu {
+  #state = {
+    products: [],
+  };
+
+  get state() {
+    return this.#state;
+  }
+
+  set state(state) {
+    this.#state = state;
+  }
+
+  init() {
+    this.#state = {
+      products: JSON.parse(localStorage.getItem('products')) ?? [],
+    };
+    this.#render();
+    this.#renderState();
+    this.#bindEvents();
+  }
+
+  #resetInput() {
+    $(SELECTOR.PRODUCT_NAME_INPUT).value = '';
+    $(SELECTOR.PRODUCT_PRICE_INPUT).value = '';
+    $(SELECTOR.PRODUCT_QUANTITY_INPUT).value = '';
+  }
+
+  #handleProductAddButtonClick() {
+    try {
+      const name = $(SELECTOR.PRODUCT_NAME_INPUT).value;
+      validateProductName(name);
+
+      const price = $(SELECTOR.PRODUCT_PRICE_INPUT).value;
+      validateProductPrice(price);
+
+      const quantity = $(SELECTOR.PRODUCT_QUANTITY_INPUT).value;
+      validateProductQuantity(quantity);
```

Comment:
검증하기전에 타입을 변경해주는게 나을 것 같아요 👀 
`<input type='number' />`의 경우 valueAsNumber를 사용해도 안전하다고 생각해요
```suggestion
      const price = $(SELECTOR.PRODUCT_PRICE_INPUT).valueAsNumber;
      validateProductPrice(price);

      const quantity = $(SELECTOR.PRODUCT_QUANTITY_INPUT).valueAsNumber;
      validateProductQuantity(quantity);
```

Reply:
저도 검증하기전에 형변환을 해주는 것이 좋을 것 같습니다. 
지금의 경우에는 문자열인 상태로 검증을 하고 있는데 자동 형변환이 되는 연산자를 이용하여 검증이 되어서 별 문제가 없었던 것 같네요.
또한 `<input type='number' />`이기 때문에 valueAsNumber를 사용하는 것이 좋다고 생각됩니다!